x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/configure_pipeline.tsx

```diff
@@ -78,7 +78,7 @@ export const ConfigurePipeline: React.FC = () => {
               'xpack.enterpriseSearch.content.indices.pipelines.addInferencePipelineModal.steps.configure.name.helpText',
               {
                 defaultMessage:
-                  'Pipeline names can only contain letters, numbers, underscores, and hyphens. The pipeline name will be automatically prefixed with "ml-inference-".',
+                  'Pipeline names must be unique across the entire deployment and can only contain letters, numbers, underscores, and hyphens. The pipeline name will be automatically prefixed with "ml-inference-".',
```

Comment:
Non-blocking thought:

`'Pipeline names must be unique across the entire deployment and can only contain letters, ...`

What would you think about something like:

`'Pipeline names are unique within a deployment and can only contain letters, ...`